### PR TITLE
Update docs to include missing details regarding FFE re-build with new changes

### PR DESCRIPTION
**Pull request by nvukobratTT**

### CODE REVIEW

File: docs/src/build.md [hunk 1]
```diff
@@ -103,8 +103,13 @@ We provide two Docker images for tt-forge-fe:
...
 2. **Prebuilt Environment Image**: This image also comes with a prebuilt environment, allowing you to skip the environment build step.
 
-
-```
+```sh
 ghcr.io/tenstorrent/tt-forge-fe/tt-forge-fe-base-ird-ubuntu-22-04
 ghcr.io/tenstorrent/tt-forge-fe/tt-forge-fe-ird-ubuntu-22-04
 ```
+
+_Note: Before re-building Forge FE with new changes, make sure to update access rights FFE and MLIR toolchains:_
```
Comment (pilkicTT) [inline docs/src/build.md:111]: Is this related only for docker images? I think we should be more clear, e.g.:

_Note: To be able to build `tt-forge-fe` inside the docker containers, make sure to set yourself as the owner of `tt-forge-fe` and `tt-mlir` toolchain directories:_

Comment (nvukobratTT) [reply docs/src/build.md:111]: Yup, only to docker images. 

There is already a section which covers these:

image

However, being more explicit is a valid point :))

**Review by pilkicTT — APPROVED** [review]
